superset-frontend/src/dashboard/util/charts/getEffectiveExtraFilters.ts

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-export default function getEffectiveExtraFilters(filters) {
+export default function getEffectiveExtraFilters(filters: Record<string, any>) {
```

Comment:
There is `DataRecordFilters` in `@superset-ui/chart` that should match `filters` here. And it'd be good to also create an `ExtraFilter` type:

```ts
import { DataRecordFilters, DataRecordValue } from "@superset-ui/chart";

interface ExtraFilter {
  col: string;
  op: 'in' | 'eq' | ...,
  val: DataRecordValue[]
}

export default function getEffectiveExtraFilters(filters: DataRecordFilters): ExtraFilter[] {
```

Reply:
Updated, TS can automagically infer the return type, so ExtraFilter wasn't required

Reply:
It's more for the use in other functions, but up to you.